### PR TITLE
MF-1644 - Create Postman collections for API documentation

### LINKED ISSUE #1644: Create Postman collections for API documentation
**FEATURE REQUEST**

1. Is there an open issue addressing this request? If it does, please add a "+1" reaction to the
   existing issue, otherwise proceed to step 2.

2. Describe the feature you are requesting, as well as the possible use case(s) for it.

Create Postman collections for API documentation.

4. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
Nice-to-have

### PR DESCRIPTION
**Pull request by EddahMurrey**
Signed-off-by: atrox <edamurrey@outlook.com>

### What does this do?
Adds mainflux postman collection
### Which issue(s) does this PR fix/relate to?
Put here `Resolves #XXX` to auto-close the issue that your PR fixes (if such)
Resolves #1644 
### List any changes that modify/break current functionality
None
### Notes
N/A

### CODE REVIEW

**Review by drasko — CHANGES_REQUESTED** [review]
@EddahMurrey please rename directory to `postman`, it is simpler (no need for `collection`).

Additionally - this looks good, but it also looks huge. Can we break this into different JSON files, one per microservice?

Also, if we break it like this - will we be capable to load multiple of those at the same time in Postman (not have to load one by one, where pone would remove previous)?

@dborovcanin looks like Postman can easily import multiple Swagger files and create collection: https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-api-specifications. If this is the case, do we even need Postman collections in the repo (since it can be easily generated from Swagger files)?

**Comment by EddahMurrey** [comment]
@drasko I think we can be able to load multiple files at the same time. Did you mean like the below picture?
Screenshot from 2022-08-17 12-34-43

**Comment by drasko** [comment]
> @drasko I think we can be able to load multiple files at the same time. Did you mean like the below picture?

Yes - exactly like this.

Then, I would definitely propose that we break this big collection into smaller ones - one per microservice used in Mainflux.

**Review by drasko — CHANGES_REQUESTED** [review]
This is good, just a few corrections:
1. We use the plural in the service name `things`, `users`, etc...
2. Some of the API functionalities are part of another microservice and should be in the same collection - for example `channels` are part of `things` microservice

File: api/postman/groups.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (drasko) [inline api/postman/groups.json]: I think Groups and API Key are currently part of Auth microservice

**Review by drasko — CHANGES_REQUESTED** [review]

**Comment by dborovcanin** [comment]
> @dborovcanin looks like Postman can easily import multiple Swagger files and create collection: https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-api-specifications. If this is the case, do we even need Postman collections in the repo (since it can be easily generated from Swagger files)?

@drasko @EddahMurrey Since that's the case, we probably do not need a separate collection(s). It will only make it more complex to maintain and keep them in sync. However, in that case, we should revisit and update OpenAPI docs and add examples to them - so Postman can generate nice and usable out-of-box collections.

**Comment by mteodor** [comment]
will this be used for testing?

**Comment by dborovcanin** [comment]
> will this be used for testing?

Since we already have Openapi files, and they can be imported into Postman, we decided to improve the examples there and drop this pull request.

**Pull request closed without merging**